locales/locale_en.po

```diff
@@ -25,6 +22,30 @@ msgstr ""
 msgid "Back"
 msgstr ""
 
+msgid "Music"
+msgstr ""
+
+msgid "Mute/Unmute"
+msgstr ""
+
+msgid "Sfx"
+msgstr ""
+
+msgid "Language"
+msgstr ""
+
+msgid "English"
+msgstr ""
```

Comment:
The language names should appear in their original names.

For example, imagine if we were to add Chinese support, a non-Chinese speaking user would not be able to know which language is which if they switch to Chinese.

Reply:
that makes sense

Reply:
fixing it now